mycodo/controllers/controller_trigger.py

```diff
@@ -121,28 +121,31 @@ def loop(self):
             check_approved = False
 
             # Check if the trigger period has elapsed
-            if self.trigger_type in ['trigger_sunrise_sunset',
-                                     'trigger_run_pwm_method']:
+            if self.trigger_type == 'trigger_sunrise_sunset':
                 while self.running and self.timer_period < time.time():
                     self.timer_period = calculate_sunrise_sunset_epoch(self.trigger)
 
-                if self.trigger_type == 'trigger_run_pwm_method':
```

Comment:
This was also executed for run_pwm_method and messed up its period. Have split up this into two separate if branches